### DISCUSSION TITLE
Deterministic float calculations

### DISCUSSION THREAD

**Discussion by Krakean** [mathnet/mathnet-numerics#819] [Q&A] [answered]
Hello.
Is Math.Net support deterministic calculations for floats (basically, restrict precision / fixed point)? Its specifically required to make a game like an RTS networked.
Some reading, in case if someone not familiar with the subject - http://gafferongames.com/networking-for-game-programmers/floating-point-determinism/

**Comment by cdrnet** [comment, marked as answer]
No, it least not that I'm aware of. The MKL provider offers some control through the `MklConsistency` enum e.g. to guarantee that all CPUs with at least SSE2 support will return consistent and thus deterministic results, but that only applies when we actually leverage the provider.
